modules/loaders/flake.nix

```diff
@@ -22,9 +28,13 @@ in
 
       load = input:
         let
-          value = import compat { src = builtins.dirOf "${input.src}/${input.settings.target}"; };
+          value = compat.load {
+            src = builtins.dirOf "${input.src}/${input.settings.target}";
+
+            replacements = input.settings.inputs;
```

Comment:
This is using the new flake-compat located here: https://github.com/nilla-nix/flake-compat

Notably, it only supports lock files with v5+. Missing lock files are not supported.